src/content/CloudBoilerplateNet/Services/CachedDeliveryClient.cs

```diff
@@ -287,7 +359,13 @@ public async Task<TaxonomyGroup> GetTaxonomyAsync(string codename)
         {
             var identifierTokens = new List<string> { KenticoCloudCacheHelper.TAXONOMY_GROUP_SINGLE_IDENTIFIER, codename };
 
-            return await CacheManager.GetOrCreateAsync(identifierTokens, () => DeliveryClient.GetTaxonomyAsync(codename), GetTaxonomySingleDependency, ProjectOptions.CreateCacheEntriesInBackground);
+            // The "productionValidator" delegate does not have to do anything since a DeliveryException is thrown instead.
```

Comment:
I'd get rid of all those comments. They are specific for one case and they wouldn't be understood by newcomers.